src/types/JobListing.ts

```diff
@@ -19,3 +19,28 @@ export type JobTransformed = Omit<Job, 'new' | 'featured'> & {
   isFeatured: Boolean;
   tags: string[];
 };
+
+export type JobData = {
+  jobData: JobTransformed[];
+  searchFilters: string[];
+  filteredJobListings: JobTransformed[];
+};
+
+export enum SearchFilterActions {
+  AddFilter = 'ADD_FILTER',
+  RemoveFilter = 'REMOVE_FILTER',
+  ClearFilter = 'CLEAR_FILTER'
+}
+
+export type SearchFilterAction =
+  | {
+      type: 'ADD_FILTER';
```

Comment:
These strings could be replaced by the enum above, so that we can be sure there is no typo:


```suggestion
      type: SearchFilterActions.AddFilter;
```

Reply:
Done